pynest/nest/server/hl_api_server.py

```diff
@@ -674,5 +676,23 @@ def merge_dicts(response):
     return result
 
 
+def merge_response(response: list):
+    if "events" in response[0]["data"]:
+        events = [res["data"]["events"] for res in response]
+        merged = [_merge_event([e[idx] for e in events]) for idx in range(len(events[0]))]
+        return [{"data": {"events": merged}}]
+    else:
+        return response
+
+
+def _flatten(xss):
+    return [x for xs in xss for x in xs]
+
+
+def _merge_event(event: list):
+    eventKeys = list(set(_flatten([e for e in event])))
```

Comment:
I am a bit confused here. According to the type annotation in the parameter list, `event` is a `list`. What then is the purpose of the list comprehension here?